Sources/Tabs/WalletCoordinator.swift

```diff
@@ -34,7 +35,8 @@ final class WalletCoordinator: NSObject,
                         with: root,
                         networkClient: networkClient,
                         analyticsService: analyticsCenter.analyticsService,
-                        persistentSecureStore: userStore.openStore)
+                        persistentSecureStore: userStore.openStore,
+                        localAuthService: DummyLocalAuthService(context: LAContext()))
```

Comment:
Should we have the context as a convenience initialiser in DummyLocalAuthService for now instead of passing it through? Appreciate this is temporary so maybe not a worthwhile change

Reply:
Yep, good shout 🙂